tot/models/models_darts.py

```diff
@@ -13,11 +13,13 @@
 
 # check import of implemented models and consider order of imports
 try:
+    from sklearn.ensemble import RandomForestRegressor
```

Comment:
@ankke What is your opinion on importing the specific models from darts?
I think we should follow a clear way here of either importing all of them and proposing a child class or handing over the import part to the user (as in your notebook).